src/components/Steps/Map/Slidedown.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SlideDown } from 'react-slidedown';
+import 'react-slidedown/lib/slidedown.css';
+
+const Slidedown = (props) => {
+    const {
+        title, data, closed, onClick,
+    } = props;
+    return (
+        <div className="slidedown-container">
+            <span
+                onClick={onClick}
+            >
+                {title}
+            </span>
+            <SlideDown closed={closed} className="my-dropdown-slidedown">
```

Comment:
I think that closed is not really a good, verbose prop... maybe You can pass active or open and just negate it ?